pynotify/handlers.py

```diff
@@ -3,8 +3,8 @@
 from django.core.exceptions import ImproperlyConfigured
 from django.utils.functional import cached_property
 
-from .helpers import register
-from .models import AdminNotificationTemplate, Notification, NotificationTemplate
+from .helpers import get_notification_model, register
```

Comment:
Hmm, this may lead to circular imports if users would like to base their Notification model on this Notification model.

Reply:
I've split `models.py` into:
```
models/__init__.py
models/notification.py
```